### PR TITLE
Updated interop statements for 1.1

### PR DESCRIPTION
**Pull request by stuclem**
- Fixes https://github.com/vmware/vic-product/issues/41
- Fixes https://github.com/vmware/vic-product/issues/89
- Fixes https://github.com/vmware/vic-product/issues/84
- Fixes https://github.com/vmware/vic-product/issues/87

### CODE REVIEW

**Comment by stuclem** [comment]
@mhagen-vmware I think that you're best qualified to look at this globally. 

@hmahmood can you please look at HA and vApp restart?

@gigawhitlocks can you please look at maintenance mode?

Thanks!

**Comment by stuclem** [comment]
@mhagen-vmware the latest version of the full interop topic is here:
https://stuclem.github.io/vic-product/assets/files/html/1.1/vic_vsphere_admin/interop.html

Do we need to add anything specific for Registry/Management Portal? I'm thinking that we probably don't, but I might be wildly deluded about that....

File: docs/user_doc/vic_vsphere_admin/interop.md [hunk 1]
```diff
@@ -16,25 +15,27 @@ Your organization could use VMware vRealize Automation to provide a self-provisi
...
-- When you put a host into maintenance mode, vSphere Integrated Containers Engine powers off container VMs with open sessions. Container VMs that do not have open sessions are migrated to other hosts by vMotion. 
+In a cluster with fully automated DRS, if you put a host into maintenance mode, DRS migrates the VCHs and container VMs to another host in the cluster. Putting hosts into maintenance mode requires manual intervention in certain circumstances:
+
+- If VCHs and container VMs are running on a standalone ESXi host, you must power off the VCHs and container VMs before you put the host into maintenance mode.
+- If container VMs have active `docker attach` sessions, you cannot put the host into maintenance mode until the `attach` sessions end. 
 
 ## VMware vSAN&trade;
-VCHs maintain filesystem layers inherent in container images by mapping to discrete VMDK files, all of which can be housed in shared vSphere datastores, including vSAN, NFS, and iSCSI datastores.
+VCHs maintain file system layers inherent in container images by mapping to discrete VMDK files, all of which can be housed in shared vSphere datastores, including vSAN, NFS, and iSCSI datastores.
```
Comment (mhagen-vmware) [inline docs/user_doc/vic_vsphere_admin/interop.md]: we support FC now as well. Otherwise LGTM.

**Comment by gigawhitlocks** [comment]
I confirmed that the behavior I was unsure about does match the documentation that you've provided here.

LGTM :)

File: docs/user_doc/vic_vsphere_admin/interop.md (hunk 1 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/interop.md]: Added Fibre Channel to the list. Thanks @mhagen-vmware.

**Comment by stuclem** [comment]
Thanks @gigawhitlocks.

**Pull request merged**